app/models/krikri/original_record.rb

```diff
@@ -42,7 +42,8 @@ def load(identifier)
           record = new(identifier)
         end
 
-        raise "No #{self} found with id: #{identifier}" unless record.exists?
+        raise NameError, "No #{self} found with id: #{identifier}" unless 
```

Comment:
I'm not 100% sure about `NameError`, here. See: http://ruby-doc.org/core-2.2.0/NameError.html

Open to other suggestions, but it seems important to have an error other than `RuntimeError` for this.